src/storybook/stand-alone-documentaion/welcome/contributor/contributor.jsx

```diff
@@ -1,17 +1,30 @@
 import cx from "classnames";
 import { InformationBox } from "../../../components/information-box/information-box";
-import { elementColorsNames } from "../../../../utils/colors-vars-map";
+import { Link } from "../../../components";
+import { elementColorsNames } from "utils/colors-vars-map";
 import "./contributor.scss";
+import { Flex, IconButton } from "components";
+import { Email } from "../../../../components/Icon/Icons";
 
 const BASE_CLASS = "monday-storybook-welcome-contributor";
-export const Contributor = ({ imgSrc, className, fullName, title }) => {
+
+export const Contributor = ({ imgSrc, className, fullName, title, email }) => {
+  const contributorTitle = (
+    <Flex justify={Flex.justify.SPACE_BETWEEN}>
+      <h4 className="monday-storybook-information-box_title">{fullName}</h4>
```

Comment:
so information box title does not include any class name but contributor title inside information box include class name of information box?(monday-storybook-information-box_title)
it's looks like a mistake?